content/en/docs/setup/production-environment/index.md

```diff
@@ -0,0 +1,256 @@
+*workload resources*.
+
+## Production control plane
+
+The simplest Kubernetes cluster has the entire control plane and worker node
+services running on the same machine. You can grow that environment by adding
+worker nodes, as reflected in the diagram illustrated in
+[Kubernetes Components](/docs/concepts/overview/components/).
+If the cluster is meant to be available for a short period of time, or can be
+discarded if something goes seriously wrong, this might meet your needs.
+
+If you need a more permanent, highly available cluster, however, you should
+consider ways of extending the control plane. By default, one-machine control
+plane services are not highly available. If keeping the cluster up and running
+and ensuring that it can be repaired if something goes wrong is important,
+consider these steps::
+
+- *Choose deployment tools*: You can deploy a control plane using tools such
+as kubeadm, kops, and kubespray. Links from [Installing Kubernetes with deployment tools](/docs/setup/production-environment/tools/)
+contains tips for production-quality deployments using each of those deployment
+methods. Different [Container Runtimes](/docs/setup/production-environment/container-runtimes/)
+are available to use with your deployments.
+- *Manage certificates*: Secure communications between control plane services
+are implemented using certificates. These can be automatically generated
+during deployment or you can generate them using your own certificate authority.
+See [PKI certificates and requirements](/docs/setup/best-practices/certificates/) for details.
+- *Configure load balancer for apiserver*: External access to the cluster
+should be to a load balancer that can distribute requests to the apiserver service
+on different nodes. See 
+[Create an External Load Balancer](/docs/tasks/access-application-cluster/create-external-load-balancer/)
+for details.
+- *Separate and backup etcd service*: The etcd services can either run on the
+same machines as other control plane services or run on separate machines, for
+extra security and availability. Because etcd stores cluster configuration data,
+backing up the etcd database should be done regularly to ensure that you can
+repair that database if needed.
+An etcd cluster should have an [odd number of members](https://etcd.io/docs/next/faq/#why-an-odd-number-of-cluster-members)
+See [Operating etcd clusters for Kubernetes](/docs/tasks/administer-cluster/configure-upgrade-etcd/)
+and [Set up a High Availability etcd cluster with kubeadm](/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm/)
+for details.
+- *Create multiple control plane nodes*: For high availability, the
+control plane should not be limited to a single node. If the control plane
+services are run by an init service (such as systemd), it should run on at
+least three nodes. However, running control plane services as pods in
+Kubernetes ensures that if the node running one of those services goes down
```

Comment:
lines 114-116, Does this sentence add value here? There are other sections in the docs that describe Services.
nit: Rephrase, "will simply be restarted"

Reply:
I think it adds value, but I can trim the sentence. The idea is that with the init method, if one of three hosts go down, you're down to two services. With Kubernetes, if a host goes down, the third service will automatically come back up on another host. The wording is a bit clumsy. I changed that sentence to read: "However, running control plane services as pods in Kubernetes ensures that the replicated number of services that you request will always be available."